test/export/test_torchbind.py

```diff
@@ -16,6 +19,66 @@
 
 @skipIfTorchDynamo("torchbind not supported with dynamo yet")
 class TestExportTorchbind(TestCase):
+    def setUp(self):
+        @torch._library.impl_abstract_class("_TorchScriptTesting::_Foo")
```

Comment:
Naming: probably just call it "register_fake_class". I'm planning on changing `impl_abstract` to "register_fake"